routing/result_interpretation.go

```diff
@@ -367,7 +367,19 @@ func (i *interpretedResult) processPaymentOutcomeUnknown(route *route.Route) {
 // failNode marks the node indicated by idx in the route as failed. This
 // function intentionally panics when the self node is failed.
 func (i *interpretedResult) failNode(rt *route.Route, idx int) {
+	// Mark the node as failing.
```

Comment:
`idx-1` can be lifted to a new local variable, say `targetNode` perhaps.

Reply:
Technically it could, both `idx-1` expression don't have the exact same meaning. It is an index into the route hops list and a route channel index.